recipes/labmaze/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "labmaze" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/deepmind/labmaze/archive/6d7e8f058428025cd4253f1ef6a1ed6618d9b0ea.zip
+  sha256: dbd3e0b96e1b50936846bdc6f8f63d1199ca6319aefa64c0f5c9a5d29858f605
```

Comment:
https://github.com/deepmind/labmaze/issues/5

Reply:
I updated the recipe to be more clear in https://github.com/conda-forge/staged-recipes/pull/21512/commits/0c0c674e296fc03fcd3200b4e3ad62554fef73c1 .